src/app/shared/components/city-confirmation/city-confirmation.component.scss

```diff
@@ -1,34 +1,40 @@
 .city-confirmation {
   position: absolute;
-  width: 280px;
-  min-height: 100px;
+  color: #444444;
+  width: 300px;
+  min-height: 130px;
   border-radius: 10px;
   box-shadow: 0 0 10px 1px rgb(50 50 50 / 30%);
-  background: hsla(0,0%,100%,.92);
+  background: rgb(255,255,255);
```

Comment:
is any reason against using hex instead?

Reply:
fixed